README.md

```diff
@@ -43,7 +43,14 @@ server (at least an empty configuration block int the page block), other options
 sent by the client are ignored and will cause a warning.
 
 Generic query params can be added by using the `query` block. This may be useful
-if parameters should be validated which cannot be assigned to other predefined option blocks.
+if parameters should be validated which cannot be assigned to other predefined
+option blocks.
+
+A `type` param can be passed in the `body` block, or the `resource` block in
+multipart requests (like `question` in the example below).
```

Comment:
A link to jump directly to the multipart requests part of this readme would be nice, i.e.

```markdown
[multipart requests](#multipart-requests).
```